marvin_general_actions.py

```diff
@@ -16,6 +16,9 @@
 # Configuration loaded
 CONFIG = None
 
+# Have marvin said good morning today?
+todaysDate = datetime.date.today()
```

Comment:
if you rename the variable to "lastDateGreeted" or something, it makes it obvious what the variable is for everywhere, not just here where the comment is (and we can remove the comment if we rename it).